### PR TITLE
chore(ids-api): Add nx targets for ids development.

### PR DESCRIPTION
**Pull request by valurefugl**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/auth/ids-api/project.json [hunk 1]
```diff
@@ -126,6 +126,40 @@
         "parallel": false,
         "cwd": "libs/auth-api-lib"
       }
+    },
+    "ids-get-secrets": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
```
Comment (saevarma) [inline apps/services/auth/ids-api/project.json:133]: I find this to greedy, as it will completely overwrite the dev´s current `.env.secret`.

Comment (valurefugl) [reply apps/services/auth/ids-api/project.json:133]: You're not wrong. Fixed.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**